integration-cli/docker_cli_run_unix_test.go

```diff
@@ -322,9 +322,18 @@ func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
 func (s *DockerSuite) TestRunWithSwappinessInvalid(c *check.C) {
 	testRequires(c, memorySwappinessSupport)
 	out, _, err := dockerCmdWithError("run", "--memory-swappiness", "101", "busybox", "true")
-	if err == nil {
-		c.Fatalf("failed. test was able to set invalid value, output: %q", out)
+	c.Assert(err, check.NotNil)
+	expected := "Valid memory swappiness range is 0-100"
+	if !strings.Contains(out, expected) {
```

Comment:
this can be `c.Assert(out, check.Equals, expected, check.Commentf("Expected output to contain %q, not %q", out, expected)`